src/session_impl.cpp

```diff
@@ -6670,6 +6646,20 @@ namespace aux {
 
 		m_alerts.emplace_alert<dht_pkt_alert>(pkt, len, d, node);
 	}
+
+	bool session_impl::should_log_portmap(int map_transport) const
+	{
+		TORRENT_UNUSED(map_transport);
```

Comment:
instead of using this macro, either comment out the name of the argument, or remove it